hack/get_latest_python_patch.py

```
@@ -27,15 +28,11 @@ def handle_starttag(self, tag, attrs):
-                                    self.best_found_version = point
-                                    return
+                                self.candidates.append(point)
+                                return
                 except Exception:
                     pass
 
-        def get_best_version(self):
-            return self.best_found_version
-
     try:
         with urllib.request.urlopen(URL) as response:
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)